.circleci/config.yml

```diff
@@ -870,6 +870,16 @@ jobs:
           path: current_release/
       - run: IMAGE_NAME=$WATCHER_IMAGE_NAME sh .circleci/ci_publish.sh
 
+  publish_perf:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      PERF_IMAGE_NAME: "omisego/perf"
```

Comment:
just curious do we want to start migrate the naming to omgnetwork?

Reply:
I'm not sure. All dockers images have the same prefix
cc @achiurizo @InoMurko

Reply:
I'm not aware of a way to migrate a docker hub account to a new namespace! (which is what `omisego` is)